.github/dependabot.yml

```diff
@@ -8,3 +8,11 @@ updates:
   reviewers:
   - apupier
   - joshiraez
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: weekly
```

Comment:
choose weekly as upgrades are happening less often and it is the value provided as an example for it